README.md

```diff
@@ -173,24 +175,60 @@ For convenience, the bot will look for an open ART pull request and cherry-pick
 
 ## Lifecycle hooks
 
-User created scripts can be set up to be executed at specific points of the bot's lifecycle. 
-Specify a path to the script you want to run using the following parameters. These scripts can be any executable file.
+User-provided scripts can be configured to execute at specific points of the bot's lifecycle. 
+You can specify a path to the script you want to run using the following parameters. These scripts can be any executable file. The same lifecycle hook arguments can be specified multiple times to attach multiple scripts to the same hook. If multiple scripts are attached to the same lifecycle hook, they will execute in the order they are provided.
 
 - **`--pre-rebase-hook`**: Executed when repository is setup with `rebase` branch checked out on `source/branch` before rebase.
 - **`--pre-carry-commit-hook`**: Executed before carrying each commit during the rebase process. The upstream is merged into the rebase branch.
 - **`--post-rebase-hook`**: Executed after the rebase process is completed.
 - **`--pre-push-rebase-branch-hook`**: Executed before pushing the rebase branch to the remote repository.
 - **`--pre-create-pr-hook`**: Executed before creating the pull request.
 
-### Scripts stored inside the repository
+### Script sources
+
+Scripts can be loaded from local filesystem, from one of the remotes, or from the builtin scripts directory.
+
+#### Local file scripts
+
+Local file scripts can be specified either by their absolute path or by their relative path to the current working directory.
+
+##### Example
+
+```sh
+rebasebot ... \
+    --pre-rebase-hook /home/user/script.sh \
+    --pre-rebase-hook script.sh
```

Comment:
In the case of multiple scripts specified for the same hook, is the order in which they are specified also always the order of execution?

Reply:
see "If multiple scripts are attached to the same lifecycle hook, they will execute in the order they are provided." above.